docs/user-guide/dream/dream-advanced-powder-reduction.ipynb

```diff
@@ -0,0 +1,480 @@
+    "workflow[Filename[VanadiumRun]] = dream.data.simulated_vanadium_sample()\n",
+    "workflow[Filename[BackgroundRun]] = dream.data.simulated_empty_can()\n",
+    "workflow[CalibrationFilename] = None\n",
+    "\n",
+    "workflow[MonitorFilename[SampleRun]] = dream.data.simulated_monitor_diamond_sample()\n",
+    "workflow[MonitorFilename[VanadiumRun]] = dream.data.simulated_monitor_vanadium_sample()\n",
+    "workflow[MonitorFilename[BackgroundRun]] = dream.data.simulated_monitor_empty_can()\n",
+    "workflow[CaveMonitorPosition] = sc.vector([0.0, 0.0, -4220.0], unit=\"mm\")\n",
+    "\n",
+    "workflow[dream.InstrumentConfiguration] = dream.InstrumentConfiguration.high_flux\n",
+    "# We drop uncertainties where they would otherwise lead to correlations:\n",
+    "workflow[UncertaintyBroadcastMode] = UncertaintyBroadcastMode.drop\n",
+    "# Edges for binning in d-spacing:\n",
+    "workflow[DspacingBins] = sc.linspace(\"dspacing\", 0.0, 2.3434, 201, unit=\"angstrom\")\n",
+    "\n",
+    "# Do not mask any pixels / voxels:\n",
+    "workflow = powder.with_pixel_mask_filenames(workflow, [])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "26",
+   "metadata": {},
+   "source": [
+    "Now, we [map](https://scipp.github.io/sciline/user-guide/parameter-tables.html) the workflow over the desired detector names to apply it to each bank separately.\n",
+    "We could do this at some intermediate step, but it is easiest to map the final result.\n",
+    "Finally, we stack the data arrays for the individual detectors into a single data array."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "27",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "detector_names = [\"mantle\", \"endcap_forward\", \"endcap_backward\", \"high_resolution\"]\n",
+    "mapped = workflow[IofTof].map({NeXusDetectorName: detector_names})\n",
+    "workflow[IofTof] = mapped.reduce(func=powder.grouping.stack_detectors)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "28",
+   "metadata": {},
+   "source": [
+    "Now compute the result:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "29",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "result = workflow.compute(IofTof)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "30",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "result"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "31",
+   "metadata": {},
+   "source": [
+    "We can plot the detectors individually with the help of `sc.collapse`:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "32",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "split = sc.DataGroup({\n",
+    "    da.coords['detector'].value: da\n",
+    "    for da in sc.collapse(result, keep='tof').values()\n",
+    "})\n",
+    "split.hist().plot()"
```

Comment:
Some of the results look weird, with broken lines (maybe NaNs in the data?)

Reply:
> (maybe NaNs in the data?)

I think so.

But this is not a problem with the docs, AFAIK. It may be caused by the normalisation with slices the data based on the wavelength range of the monitor.

Reply:
The monitor contains NaNs. So some detector events also become NaN when normalising.

I could hide this from the basic docs by using the proton charge normalisation. Should I?